chat_app/settings.py

```diff
@@ -51,6 +52,9 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+
+    # Custom middlewares
+    'chat_logger.middleware.log_errors_to_db_middleware.Log5XXErrorsToDatabaseMiddleware',
```

Comment:
This middleware will process all the exceptions and will log them in the database.